screenpy/resolutions/is_close_to.py

```diff
@@ -16,11 +17,21 @@ class IsCloseTo:
         )
     """
 
+    @property
+    def delta_to_log(self) -> str:
+        """Represent the delta in a log-friendly way."""
+        return represent_prop(self.delta)
+
+    @property
+    def num_to_log(self) -> str:
+        """Represent the num in a log-friendly way."""
+        return represent_prop(self.num)
```

Comment:
Same with these two methods, add them to the `exclude-members` list for `IsCloseTo` in `docs/api/resolutions.rst`!